packages/core/src/logger.ts

```diff
@@ -0,0 +1,33 @@
+import * as pino from 'pino';
```

Comment:
General inquiry: does `pino` work in browser context? Remember that one of the Prism's requirements is to be able to work both in nodejs and browser (e.g. we want to use it in request maker).

It caught my attention because I wasn't sure if `core` is actually a good place to store this stuff in. It looks that `pino` is mostly CLI specific (?)

Reply:
1. Yes! http://getpino.io/#/docs/browser

2. Not entirely correct: the CLI is the place where the logs get captured from the stream and processed/visualised, but its usage spread in the whole application. Moreover, Pino implements the log4j interface so it should be, if needed, easily replaceable with anything we like in the future (Winston, Buyan). Does that clarify?